docs/architecture-decisions/2024-05-21_primitive-bytes-and-strings.md

```diff
@@ -189,8 +204,8 @@ We should select an appropriate name for the type representing an AVM string. It
    - ✅ very obvious what it is
    - ✅ obvious how it differs to `string`
 
-
+Option 5 would be the preferred option if we were to prioritize developer experience whereas option 4 would be best if we priotized control over the prototype.
```

Comment:
> prioritize developer experience

This feels a little subjective. What aspects of the developer experience are prioritized by this option? The obvious one I can see is saving a couple of characters in declaration, but at the expense of having to explicitly type variables
```
const demo: bytes = "my bytes"
vs.
const demo = Str`my bytes`

someFunc("my bytes")
vs.
someFunc(Str`my bytes`)
```
Are there any others?

Reply:
I think "a couple of characters in declaration" is a bit reductive of the impact it has on the developer experience. When developers use strings they expect to be able to put literals between quotation marks. Adding any friction to that can be a bit jarring. We saw this with PyTeal `Bytes` constructor frequently. 

That being said, I agree that "developer experience" is too subjective, so I've changed it to familiarity